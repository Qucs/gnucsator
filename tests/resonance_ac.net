# Qucs 0.0.19  /home/felix/extern/gnucap-qucs/examples/resonance.sch

Vac:V1 _net0 gnd U="1 V" f="1 GHz" Phase="0" Theta="0"
C:C1 gnd Voltage C="2 pF" V=""
L:L1 Voltage _net1 L="13 nH" I=""
R:R1 _net1 _net0 R="10" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
.AC:AC1 Type="lin" Start="0.9 GHz" Stop="1.1 GHz" Points="15" Noise="no"
