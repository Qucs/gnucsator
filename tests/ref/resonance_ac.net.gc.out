<Qucs Dataset>
default plugins: ignored
detach_all: already installed, replacing
stashing as detach_all:0
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
spice_bjt: already installed, replacing
stashing as spice_bjt:0
spice_mos: already installed, replacing
stashing as spice_mos:0
C:C1 gnd Voltage C="2 pF" V="" 
                          ^ ? C1: V has no value?
L:L1 Voltage _net1 L="13 nH" I="" 
                             ^ ? L1: I has no value?
.AC:AC1 Type="l ... .1 GHz" Points="15" Noise="no" 
                                        ^ ? what's this (incomplete)?
Gnucap   System status
iterations: op=2, dc=0, tran=0, fourier=0, total=17
transient timesteps: accepted=0, rejected=0, total=0
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=3, subckt=0, model=0, total=3
dctran density=100.0%, ac density=100.0%

#Freq                      vr(Voltage)               vr(_net0)                 vr(_net1)                 vi(Voltage)               vi(_net0)                 vi(_net1)                
 900.E+6                   4.0906882823448649233     0.9999996896270523905     0.6896267420339791343    -2.74429937841329874      -462.64594521385697944E-9 -0.46264640785980235548   
 914.28571428571441259E+6  4.2561629814798109805     0.99999960429427119379    0.60429387567851877705   -3.4441379611940590166    -489.00133870071984532E-9 -0.4890018277020582893    
 928.57142857142864755E+6  4.2844587996332439416     0.99999949253474387234    0.492534236570126005     -4.348917134916462146     -499.94375932562089559E-9 -0.49994425926937999804   
 942.85714285714288251E+6  4.0335116045002585849     0.99999935300068845123    0.35300004139548146753   -5.4606967449481471632    -477.90224459499102138E-9 -0.47790272249723564357   
 957.14285714285711748E+6  3.3095090004591942722     0.99999919743393022031    0.19743312778855502287   -6.6725858208453043829    -398.06151651984995609E-9 -0.39806191458136641614   
 971.42857142857135244E+6  1.9750741223151750958     0.99999906197260575169    0.061971667731041124582  -7.6841314400300415599    -241.10384457218598797E-9 -0.24110408567603029972   
 985.7142857142855874E+6   0.17448746194103517126    0.99999900046836227573    467.36283031123528886E-6 -8.069295698172689768     -21.61350198391200017E-9  -0.021613523597414250332  
 999.99999999999982236E+6 -1.6032866855719943188     0.99999904239002610495    0.042389068553455260257  -7.6204180445095417439     201.4749469196519982E-9   0.20147514839459912394   
 1.0142857142857142349E+9 -2.888440790521661583      0.99999916168177254149    0.1616809340634598513    -6.577164990608719286      368.15749149489596092E-9  0.36815785965238747579   
 1.0285714285714284699E+9 -3.5669551316762273529     0.99999930651063166919    0.30650993830787047755   -5.365318239708435577      461.04356725131898642E-9  0.46104402829488613235   
 1.0428571428571427048E+9 -3.7883357386917033693     0.99999944060153485025    0.44060097539041587211   -4.2686110703738675553     496.45872228480190102E-9  0.49645921874352367808   
 1.0571428571428571618E+9 -3.7443322329956760796     0.99999955078345639236    0.55078300707372296329   -3.3815218523677299522     497.41390345578384568E-9  0.49741440086968768952   
 1.0714285714285716188E+9 -3.5717483166472767663     0.9999996368784954015     0.63687813228247680541   -2.6969871786397279223     480.8990688021769877E-9   0.48089954970124555444   
 1.0857142857142856229E+9 -3.3488895205125999154     0.99999970306947219       0.70306917528988988053   -2.1763534786647440455     456.90534306559600352E-9  0.45690579997093925257   
 1.1000000000000000888E+9 -3.1157409752978564299     0.99999975398157587846    0.7539813299165770033    -1.7797749218919394209     430.6891141733159678E-9   0.43068954486242967405   
